app/shared_features/camera/VideoRecorder.js

```diff
@@ -10,9 +10,17 @@ import {
 }  from 'react-native';
```

Comment:
Use redux instead of internal state.

Pass in props via container / screen.

When you tap to open it, that's when the action should set the default camera.